src/components/annotations3d/attributes.js

```diff
@@ -75,13 +75,13 @@ module.exports = {
     arrowwidth: annAtts.arrowwidth,
     standoff: annAtts.standoff,
     hovertext: annAtts.hovertext,
-    hoverlabel: annAtts.hoverlabel
+    hoverlabel: annAtts.hoverlabel,
+    captureevents: annAtts.captureevents
```

Comment:
With this, hover labels on 3d annotations are working:
![peek 2017-05-25 16-37](https://cloud.githubusercontent.com/assets/6675409/26469595/94a5809e-4168-11e7-8ada-349db5396d5e.gif)